autoprof/fit/langevin.py

```diff
@@ -51,7 +52,24 @@ class MALA(BaseOptimizer):
 
     Borrowed heavily from Alexandre Adam: https://github.com/AlexandreAdam/
     """
-    def __init__(self, model, initial_state = None, **kwargs):
+    def __init__(self, model: 'AutoProf', initial_state: Optional[Dict[str,torch.Tensor]] = None, **kwargs)-> None:
```

Comment:
The model is  of type `AutoProf_Model`, The initial state should more likely be: `Optional[Sequence]`